src/components/Button/styles.js

```diff
@@ -78,8 +78,20 @@ const styles = {
 
     return '100%';
   },
+  marginLeft({ margin, disableAutoMarginLeft }) {
+    if (disableAutoMarginLeft) {
+      return 0;
+    }
+    if (!margin) {
+      return theme.space.m;
+    }
+    return undefined;
+  },
```

Comment:
Looks good for the disabled version. But I see you set a default `space.m` value if no margin is specified, but you never use this `margin` param if set. Why?

Reply:
`margin` props is used on line 145. I kept the logic as it was, to only reset the margin-left for the second button that needs margin-left, let me know if it's unclear :)